### PR TITLE
feat: created badges with avatar

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
Screenshot (42)

### CODE REVIEW

File: components/avatars.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@import url(/components/button.css);
+
+
+.avatar>img{
+    border: solid var(--primary-100);
+    height: auto;
+    width: 7rem;
+    border-radius: 50%;
+    text-align: center;
+
```
Comment (rahulcs754) [inline components/avatars.css]: remove space

File: components/avatars.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    border: solid var(--primary-100);
+    height: auto;
+    width: 7rem;
+    border-radius: 50%;
+    text-align: center;
+    
+}
+
+.avatar-container{
+
```
Comment (rahulcs754) [inline components/avatars.css]: same above

**Review by rahulcs754 — COMMENTED** [review]
all look good 
some changes

**Review by rahulcs754 — APPROVED** [review]

File: components/avatars.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+@import url(/stylesheets/style.css);
+@import url(/components/button.css);
+
+.avatar > img {
+  border: solid var(--primary-100);
+  height: auto;
+  width: 7rem;
+  border-radius: 50%;
+  text-align: center;
```
Comment (coldpigli) [inline components/avatars.css]: No need for text-align on images.

File: components/avatars.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+.offline {
+  background-color: rgb(193, 183, 183);
+}
+
+.busy {
+  background-color: rgb(228, 59, 59);
+}
+
+.avatar-xs {
+  width: 30%;
```
Comment (coldpigli) [inline components/avatars.css]: this code will break on a different parent container

**Review by coldpigli — CHANGES_REQUESTED** [review]
Added some request for changes via comments

File: components/avatars.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply components/avatars.css]: made required changes

File: components/avatars.css (hunk 3 above)
Comment (vaibhav-mahalle) [reply components/avatars.css]: I have made the required changes. Please approve them.

**Review by coldpigli — APPROVED** [review]

File: components/avatars.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply components/avatars.css]: done required changes

**Pull request merged**